File: mybin/se/sysconfig/formatter/cs/examples/constructor.cs
public class Foo : Bar
{
    public Foo(int cap, bool resize = false):base(cap){
        if (resize) 
            m_something = cap;
        else 
            m_something = 12;
    }

    public void Frob(int n=0) {
       _intern_frob(n, CHK); 
    }
}

File: mybin/se/sysconfig/formatter/c/examples/constructor.cpp

class Foo : public Bar
{
public:
    Foo();
    Foo(int cap, bool resize = false);
    virtual char* baz() = 0;
};

Foo::Foo()
{
}

Foo::Foo(int cap, bool resize) : m_cap(cap), m_resize(resize)
{
}
